data/hai/unfettered missions.txt

```diff
@@ -0,0 +1,3839 @@
+			choice
+				`	"You don't scare me."`
+				`	"On second thought, I'll come back later."`
+					goto later
+				`	"Now that it's here I'm starting to realize I don't want to be a part of your way. I'm out.`
+					decline
+			`	"That is good! Let us fight, on the same terms of honor as before. Take off when you're ready, I will be waiting for you. Once you defeat me you will be able to fight the Tejolis."`
+				accept
+			label later
+			`	"Be sure to come back soon. War cannot wait forever."`
+				defer
+	npc save disable
+		to despawn
+			has "unfettered: defeated Si-Ra"
+		government "Hai (Unfettered Challenge)"
+		ship "Enhanced Shield Beetle (Pulse)" "Intrepid II"
+		personality disables staying heroic target launching
+		conversation
+			`"You have defeated me, But Hi-Ye will not be as easy to beat! Land to repair, and rhen take off when you're be ready to face him."`
+			action
+				set "unfettered: defeated Si-Ra"
+	npc save disable
+		to spawn
+			has "unfettered: defeated Si-Ra"
+		government "Hai (Unfettered Challenge)"
+		ship "Shield Beetle Prototype (Black)" "Tejolis"
+		personality disables staying heroic target launching
+		conversation
+			action
+				"reputation: Hai (Unfettered)" += 100
+				event "shield beetles accessible"
+			`Now that the fight is over, they both hail you, and He-Yi speaks first. "You sure know how to fight! I feel now maybe we are the ones who need training. I feel like it is only fair you give us tips this time."`
+			choice
+				`	"You were overconfident, and that's what let you down."`
+					goto mistake
+				`	"A good fight, but you just had no chance against my superior firepower."`
+				`	"You lacked coordination with each other."`
+					goto mistake
+				`	"Nothing specific. Well fought to both of you."`
+					goto train
+			`	He-Yi answer first. "It looks like that was indeed the case. Sheer strength may be enough to defeat nearly everything in your way, but is it fun? Is it strategic? I try not to rely on that tactic too often."`
+			`	Sa-Ri then speaks. "Well captain, I sure wish I had the money for such a fleet. I could barely afford this ship, and it will need repairs again now."`
+				goto train
+			label mistake
+			`	He-Yi answers first. "Yes, that is true. We'll try not to repeat that mistake again, Captain."`
+			`	Sa-Ri then speaks. "Next time it will not be so easy, Captain!"`
+			label train
+			`	"The war will start soon, but there is still some time to train. Use that time well, Captain. We have also given you access to our mighty Shield Beetles, should you need it."`
+	on fail
+		dialog phrase "Essential Unfettered Mission Fail"
+	on complete
+		log `Officially joined the Unfettered Militia and received a license to purchase their heavy ships.`
+		"salary: Unfettered" = 5000
+		set "license: True Unfettered"
+		event "joined the unfettered"
+		clear "unfettered: defeated Si-Ra"
+		#"reputation: Wanderer" <?= -1000
```

Comment:
Why is this `#`ed out?

Reply:
For testing for this PR, I dont think I can justify it being commented once it gets merged

Reply:
Yeah, just make sure it's removed before this is merged then.

Reply:
Added a comment to be sure.